e2e/framework/README.md

```diff
@@ -0,0 +1,23 @@
+# rook-test-framework
+ 
+Requirements:
+-   Docker 1.12.6 
+-   Ubuntu Host
+ 
+Instructions 
+On your Ubuntu box do the following;
+-   Be sure to allocate atleast 4 processors to your vm
+-   git clone https://github.com/rook/rook
+-   Navigate to the rook directory of the clone sources;
+-         e2e/scripts/smoke_test.sh
+ 
+ 
+ 
+At this point a lot of stuff should be happening on your console output,
+-         create rook-infra container
+-         mount the docker.sock on the host
+-         create 3 node k8s cluster using kubeadm
+-         install ceph on k8s nodes
+-         Install rook components
+-         Run smoke tests
+-         Finally the rook-test container will get stopped
```

Comment:
Also some instructions around where to look/poke if something goes wrong? (e.g., an overview of the running containers and their responsibilities)